fastflix/widgets/panels/status_panel.py

```diff
@@ -163,6 +207,23 @@ def closeEvent(self, event):
         self.hide()
 
 
+class ElapsedTimeTicker(QtCore.QThread):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self.parent = parent
+        self.running = True
+
+    def __del__(self):
+        self.wait()
+
+    def run(self):
+        while self.running:
+            self.parent.signal_elapsed_time.emit("tick")
+            time.sleep(0.3)
```

Comment:
There is no reason that this should bother me at all, but could we make it divisible by a full second? i.e. `0.2` or `0.5` ?

Reply:
I can totally relate :grinning: Changed to `0.2` in c1e86d2